src/main/java/core/basesyntax/serviceimpl/CsvToFruitTransactionConverter.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CsvToFruitTransactionConverter implements DataConverter<List<String>,
+        List<FruitTransaction>> {
+    private static final int TYPE = 0;
+    private static final int FRUIT = 1;
+    private static final int QUANTITY = 2;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputReport) {
+        return inputReport.stream()
+                .map(line -> {
+                    String[] parts = line.split(",");
```

Comment:
There is a potential issue with splitting the CSV line. If there are any leading or trailing spaces around the commas, this will not be handled properly. It's a good practice to trim the parts after splitting to avoid such issues.